newdle/client/src/styles/palette.scss

```diff
@@ -7,3 +7,4 @@ $peach: #ffb3b8;
 $green: #62f46f;
 $light-teal: #00caa8;
 $blue: #2185d0;
+$don-juan: #60495a;
```

Comment:
LOL :es: :crossed_swords:

Reply:
It's actually the name of the color: https://huehex.com/color/64514b 😆